src/Components/PaperEditor/index.js

```diff
@@ -8,7 +8,7 @@ import Button from 'react-bootstrap/Button';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faAngleUp, faAngleDown } from '@fortawesome/free-solid-svg-icons';
 import Breadcrumb from '@bbc/digital-paper-edit-storybook/Breadcrumb';
-import Transcripts from './Transcripts';
+// import Transcripts from './Transcripts';
```

Comment:
```suggestion
import Transcripts from './Transcripts';
```
I think we should leave this uncommented when we push - we don't want to confuse the people working on the Transcript element.